### PR TITLE
Phase2-hgx327H Clean up unused obsolete python files preceding V16 of HGCAL specification from Geometry/HGCalCommonData

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Clean up unused obsolete python files preceding V16 of HGCAL specification from Geometry/HGCalCommonData

#### PR validation:

Use the test scripts in Geometry/HGCalCommonData/test/python

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
+geometry

**Comment by bsunanda** [comment]
@srimanob Please approve this

File: Geometry/HGCalCommonData/test/python/runHGCalTestDD4hep_cfg.py [hunk 1]
```diff
@@ -1,20 +1,54 @@
...
 process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
 process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.EventContent.EventContent_cff')
 process.load('SimGeneral.MixingModule.mixNoPU_cfi')
-process.load('Geometry.HGCalCommonData.testHGCalWaferFXML_cfi')
 process.load('Geometry.EcalCommonData.ecalSimulationParameters_cff')
 process.load('Geometry.HcalCommonData.hcalDDDSimConstants_cff')
-process.load('Geometry.HGCalCommonData.hgcalEEParametersInitialization_cfi')
-process.load('Geometry.HGCalCommonData.hgcalEENumberingInitialization_cfi')
+process.load('Geometry.HGCalCommonData.hgcalV15ParametersInitialization_cfi')
```
Comment (srimanob) [inline Geometry/HGCalCommonData/test/python/runHGCalTestDD4hep_cfg.py:45]: Should this file be renamed to V16?

**Comment by srimanob** [comment]
+Upgrade

This is a clean up PR, no effect on simulation and reconstruction. There is a comment on file renaming which can come in the follow up PR.

**Comment by perrotta** [comment]
+1

**Pull request merged**